crates/serpen/tests/fixtures/ast_rewriting/class_name_collision/models.py

```
@@ -0,0 +1,34 @@
+"""
+Models module with User and Product classes.
+These will conflict with entities.py when bundled.
+"""
+
+
+class User:
+    """User model for data persistence."""
+
+    def __init__(self, name: str, email: str):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)